examples/packages/parcel-resolver-alias/tsconfig.json

```diff
@@ -6,5 +6,5 @@
     "module": "commonjs"
   },
   "exclude": ["dist"],
-  "references": []
+  "references": [{ "path": "../../../packages/jest" }]
```

Comment:
Because of type checking now being enabled in all projects pretty much every project needs to reference `jest` because of the global matcher.